pkg/ini_test.go

```diff
@@ -0,0 +1,219 @@
+package main
+
+import (
+	"os"
+	"reflect"
+	"slices"
+	"testing"
+)
+
+const testParserString = `[forge.example]
+User = hg
+
+[topsecret.server.example]
+Port = 50022
+ForwardX11 = no
+`
+
+func checkError(t *testing.T, err error) {
```

Comment:
use a better name, as you are marking the test as failed not "checking something"